pulsar-client-cpp/lib/MultiTopicsConsumerImpl.cc

```diff
@@ -52,6 +53,13 @@ MultiTopicsConsumerImpl::MultiTopicsConsumerImpl(ClientImplPtr client, const std
     } else {
         unAckedMessageTrackerPtr_.reset(new UnAckedMessageTrackerDisabled());
     }
+    auto partitionsUpdateInterval = static_cast<unsigned int>(client_->conf().getPartitionsUpdateInterval());
+    if (partitionsUpdateInterval > 0) {
+        listenerExecutor_ = client_->getListenerExecutorProvider()->get();
```

Comment:
```suggestion
```

The `listenerExecutor_` has already been initialized in the member initializer list. Maybe we should add `const` before `listenerExecutor_` to avoid it being assigned a value after being initialized.

```c++
const ExecutorServicePtr listenerExecutor_;
```

Reply:
Oh, sorry. I fixed it.